src/controllers/Controller.js

```diff
@@ -300,4 +304,78 @@ async loginRestaurant(req, res) {
     }
   }
 
+  async servePaymentForm(req, res) {
+    try {
+      const filePath = './public/payment-form.html';
+      fs.readFile(filePath, (err, data) => {
+        if (err) {
+          res.writeHead(500, { 'Content-Type': 'text/plain' });
+          res.end('Failed to fetch payment-form.html ' + ' ' + err);
+        } else {
+          res.writeHead(200, { 'Content-Type': 'text/html' });
+          res.end(data);
+        }
+      });
+    } catch (error) {
+      response(res, { status: 400, data: error.message });
+    }
+  }
+
+  async getClientPaymentToken(req, res) {
+    try {
+      const filePath = './public/scripts/get-client-token.js';
+      fs.readFile(filePath, (err, data) => {
+        if (err) {
+          res.writeHead(500, { 'Content-Type': 'text/plain' });
+          res.end('Failed to fetch get-client-token.js ' + ' ' + err);
+        } else {
+          res.writeHead(200, { 'Content-Type': 'text/html' });
+          res.end(data);
+        }
+      });
+    } catch (error) {
+      response(res, { status: 400, data: error.message });
+    }
+  }
+
+  async submitPayment(req, res) {
+    try {
+      let data = '';
+
+    req.on('data', (chunk) => {
+      data += chunk;
+    });
+    req.on('end', async () => {
+      try {
+        const requestBody = JSON.parse(data);
+        
+        await stripe.charges
+          .create({
+            amount: 10000, // Amount in cents
+            currency: 'usd',
+            description: 'Example Charge',
+            source: requestBody.token,
+          })
+          .then((charge) => {
+            res.setHeader('Content-Type', 'application/json');
+            res.end(JSON.stringify({ message: 'Payment successful', charge }));
+          })
+          .catch((err) => {
+            console.log('err');
+            console.log(err);
+            res.statusCode = 500;
+            res.setHeader('Content-Type', 'application/json');
+            res.end(
```

Comment:
Can you please use the response method like down below
response(res, { status: 400, data: error.message });